samples/graph-activity-feed/csharp/ActivityFeedNotification/Views/Home/GroupChatNotification.cshtml

```diff
@@ -63,28 +68,34 @@
                     });
                 }
             });
+
             if (isValid == false) {
                 e.preventDefault();
                 return false;
             }
+
             var action = "EntityTopic";
-            let taskInfo = {
+            let taskModuleData = {
                 title: $('#title').val(),
                 description: $('#description').val(),
                 userName: username,
                 taskInfoAction: action,
                 access_token: token,
                 chatId:groupchatId,
             };
+
             $.ajax({
                 type: 'POST',
                 url: '/SendNotificationToGroupChat',
                 dataType: 'json',
-                data: taskInfo
+                data: taskModuleData
             });
-            microsoftTeams.tasks.submitTask(taskInfo);
+
+            microsoftTeams.tasks.submitTask(taskModuleData);
```

Comment:
What is this submitTask being used for? I don't see a startTask anywhere? https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/task-modules/task-modules-tabs?tabs=teamsjs1%2Cteamsjs2

Reply:
Yes its not required, removed it